go_proto_library failed to build on OS X 10.13

WORKSPACE
```
git_repository(
    name = "io_bazel_rules_go",
    remote = "https://github.com/bazelbuild/rules_go.git",
    tag = "0.5.2",
)

load("@io_bazel_rules_go//go:def.bzl", "go_repositories", "go_repository")
go_repositories()

load("@io_bazel_rules_go//proto:go_proto_library.bzl", "go_proto_repositories")
go_proto_repositories()
```

BUILD
```
load("@io_bazel_rules_go//go:def.bzl", "go_prefix")
load("@io_bazel_rules_go//proto:go_proto_library.bzl", "go_proto_library")

go_prefix("test")
go_proto_library(
    name = "useful_proto",
    srcs = ["useful.proto"],
    rules_go_repo_only_for_internal_use = "@",
    visibility = ["//visibility:public"],
    deps = ["@com_github_golang_protobuf//ptypes/duration:go_default_library"],
)
```

useful.proto from examples, after removing external dependency
```
syntax = "proto3";

package very.useful;

message VeryUseful {
  string very = 1;
  int32 useful = 2;
}
```

```shell
$ bazel build :all
INFO: Found 4 targets...
ERROR: /private/var/tmp/_bazel_x/8f260ef0f9000f3e2d423d9a2e901b88/external/com_github_google_protobuf/BUILD:248:1: C++ compilation of rule '@com_github_google_protobuf//:protoc_lib' failed: Process exited with status 1 [sandboxed].
external/com_github_google_protobuf/src/google/protobuf/compiler/command_line_interface.cc:1768:28: error: out-of-line definition of 'WriteDescriptorSet' does not match any declaration in 'google::protobuf::compiler::CommandLineInterface'
bool CommandLineInterface::WriteDescriptorSet(
                           ^~~~~~~~~~~~~~~~~~
/usr/local/include/google/protobuf/compiler/command_line_interface.h:261:13: note: type of 1st parameter of member declaration does not match definition ('const std::vector<const FileDescriptor *> &' vs 'const std::vector<const FileDescriptor *>')
      const std::vector<const FileDescriptor*>& parsed_files);
            ^
1 error generated.
Use --strategy=CppCompile=standalone to disable sandboxing for the failing actions.
INFO: Elapsed time: 188.428s, Critical Path: 14.88s
```

It seems that the failure is caused by v3.2.0 github.com/google/protobuf, after manually pin to v3.3.0 the problem is resolved.

```
go_proto_repositories(shared=0)

http_archive(
    name = "com_github_google_protobuf",
    url = "https://github.com/google/protobuf/archive/v3.3.0.tar.gz",
    strip_prefix = "protobuf-3.3.0",
    sha256 = "94c414775f275d876e5e0e4a276527d155ab2d0da45eed6b7734301c330be36e",
)
```

--- a/proto/go_proto_library.bzl
+++ b/proto/go_proto_library.bzl
@@ -331,9 +331,9 @@ def go_proto_repositories(shared = 1):
     # if using multiple *_proto_library, allows caller to skip this.
     native.http_archive(
         name = "com_github_google_protobuf",
-        url = "https://github.com/google/protobuf/archive/v3.2.0.tar.gz",
-        strip_prefix = "protobuf-3.2.0",
-        sha256 = "2a25c2b71c707c5552ec9afdfb22532a93a339e1ca5d38f163fe4107af08c54c",
+        url = "https://github.com/google/protobuf/archive/v3.3.0.tar.gz",
+        strip_prefix = "protobuf-3.3.0",
+        sha256 = "94c414775f275d876e5e0e4a276527d155ab2d0da45eed6b7734301c330be36e",
     )
 
   # Needed for gRPC, only loaded by bazel if used

